nomad/structs/structs.go

```diff
@@ -10612,8 +10612,19 @@ type DeploymentStatusUpdate struct {
 // RescheduleTracker encapsulates previous reschedule events
 type RescheduleTracker struct {
 	Events []*RescheduleEvent
+
+	// LastReschedule represents whether the most recent attempt to reschedule
+	// the allocation (if any) was successful
+	LastReschedule ResheduleTrackerAnnotation
 }
 
+type ResheduleTrackerAnnotation string
```

Comment:
```suggestion
type RescheduleTrackerAnnotation string
```

(also consecutie typos wherever this type is used)

Reply:
:facepalm: